deepchem/models/tests/test_layers.py

```diff
@@ -1438,3 +1438,16 @@ def test_mxmnet_sphericalbasis():
                        [-0.0000e+00, -0.0000e+00, -0.0000e+00, -0.0000e+00]])
 
     assert np.allclose(output, result, atol=1e-04)
+
+
+@pytest.mark.torch
+def test_torch_highway_layer():
+    """Test invoking HighwayLayer."""
+    torch.manual_seed(0)
```

Comment:
do we need to fix a seed here?
If the test outputs don't pass the assertions sometimes, you can add a `@flaky` tag above the `@pytest.mark.torch`

Reply:
Removed it. Its not needed for the current test.